Marlin/src/module/motion.cpp

```diff
@@ -425,10 +425,9 @@ void _internal_move_to_destination(const_feedRate_t fr_mm_s/*=0.0f*/
     planner.e_factor[active_extruder] = 1.0f;
   #endif
 
-  if (TERN0(IS_KINEMATIC, is_fast))
-    TERN(IS_KINEMATIC, NOOP, prepare_line_to_destination());
-  else
-    prepare_line_to_destination();
+  //if (TERN0(IS_KINEMATIC, is_fast)) prepare_fast_move_to_destination();
+  if (TERN0(IS_KINEMATIC, is_fast)) prepare_internal_move_to_destination();
```

Comment:
this doesn't seems right.

Reply:
Surely, the top line that is commented may seem more correct. But it produced a compilation error for FYTSEC_S6.
All I did was follow the recommendation provided by the build. And the result, OK and G33 works again.
I think 'rq3' has shown that the error was there. But you decide, which is better and more elegant. The only thing I did is resolve the inconsistency and make it compatible with FYTSEC_S6. And I think your last post describes that you modified 'motion.cpp' based on my pull_request and it worked perfectly, also with two types of probe_z.